fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Exterior.xml

```diff
@@ -0,0 +1,201 @@
+<ModelBehaviors>
```

Comment:
Should the generated files be checked in? I suspect there's something wrong there.

Reply:
will fix

Reply:
> Should the generated files be checked in? I suspect there's something wrong there.

Are they even generated right now? Can't find a build script. Maybe leave them for now until these are really generated.

Reply:
Also there are A32NX
And there several A380_COCKPIT.xml etc. 
I think a good clean up after the move makes sense - from someone who understands all these files better ;)

Reply:
> Maybe somebody did a bit of a hack there.

Looks like Holland:
![image](https://github.com/flybywiresim/aircraft/assets/16833201/ed31215b-8e17-4109-b63c-6b7cc4d4faf9)

But also no changes ever - I do not think these are required any more. As I said - let's do a cleanup after the move.

Reply:
IIRC there were some changes done in the generated xml files, which we have to carry over to the original files.

Reply:
> IIRC there were some changes done in the generated xml files, which we have to carry over to the original files.

Are these generated files even in use (referenced)? Or are they just to have the copy template?

Reply:
> IIRC there were some changes done in the generated xml files, which we have to carry over to the original files.

Damn, that sounds even worse 😬

Reply:
> > IIRC there were some changes done in the generated xml files, which we have to carry over to the original files.
> 
> Are these generated files even in use (referenced)? Or are they just to have the copy template?

They are used.

Reply:
> They are used.

Then we keep them for now of course until this is cleaned up.

Ready to merge if everybody else is good with it.